src/components/application_manager/src/request_controller.cc

```diff
@@ -190,8 +190,9 @@ RequestController::TResult RequestController::addHMIRequest(
 
   const uint64_t timeout_in_mseconds =
       static_cast<uint64_t>(request->default_timeout());
-  RequestInfoPtr request_info_ptr(
-      new HMIRequestInfo(request, timeout_in_mseconds));
+  RequestInfoPtr request_info_ptr =
+      utils::MakeShared<HMIRequestInfo>(request, timeout_in_mseconds);
+  request_info_ptr->set_hmi_level(mobile_apis::HMILevel::INVALID_ENUM);
```

Comment:
@anosach-luxoft After I failed to reproduce the bad_alloc() exception I decided to do a check with a `valgrind` in order to reveal hidden memory access problems which may cause the observed crash. I discovered usage of uninitialized variable and after a consultation with @dev-gh I added the discussed line to which corresponds the `Fixed uninitialized values in request_controller.` line in description of commit. I am not sure if this may cause the observed crash but definitely this is a bug and it should be fixed.